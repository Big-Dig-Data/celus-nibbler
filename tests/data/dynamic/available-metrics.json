{
  "kind": "non_counter.date_based",
  "version": 1,
  "parser_name": "available-metrics",
  "data_format": {
    "name": "simple_format"
  },
  "platforms": [
    "Platform1",
    "Platform2"
  ],
  "metrics_to_skip": [],
  "available_metrics": ["deny"],
  "titles_to_skip": [],
  "dimensions_to_skip": {},
  "metric_aliases": [
    ["Denied", "deny"]
  ],
  "dimension_aliases": [],
  "heuristics": {
    "conds": [
      {
        "pattern": "Title",
        "coord": {
          "row": 0,
          "col": 0
        },
        "kind": "regex"
      },
      {
        "pattern": "ISSN",
        "coord": {
          "row": 0,
          "col": 1
        },
        "kind": "regex"
      },
      {
        "pattern": "Platform",
        "coord": {
          "row": 0,
          "col": 2
        },
        "kind": "regex"
      },
      {
        "pattern": "YOP",
        "coord": {
          "row": 0,
          "col": 3
        },
        "kind": "regex"
      },
      {
        "pattern": "Metrics",
        "coord": {
          "row": 0,
          "col": 4
        },
        "kind": "regex"
      }
    ],
    "kind": "and"
  },
  "areas": [
    {
      "data_headers": {
        "roles": [
            {
                "source": {
                    "coord": {
                      "row": 0,
                      "col": 5
                    },
                    "direction": "right"
                },
                "role": "date"
            }
        ],
        "data_direction": "down",
        "data_cells": {
          "coord": {
            "row": 1,
            "col": 5
          },
          "direction": "right"
        }
      },
      "titles": {
        "source": {
          "coord": {
            "row": 1,
            "col": 0
          },
          "direction": "down"
        }
      },
      "metrics": {
        "source": {
          "coord": {
            "row": 1,
            "col": 4
          },
          "direction": "down"
        }
      },
      "title_ids": [
        {
          "name": "Print_ISSN",
          "source": {
            "coord": {
              "row": 1,
              "col": 1
            },
            "direction": "down"
          }
        }
      ],
      "dimensions": [
        {
          "name": "YOP",
          "source": {
            "coord": {
              "row": 1,
              "col": 3
            },
            "direction": "down"
          }
        },
        {
          "name": "Platform",
          "source": {
            "coord": {
              "row": 1,
              "col": 2
            },
            "direction": "down"
          }
        }
      ],
      "kind": "non_counter.date_based"
    }
  ]
}
