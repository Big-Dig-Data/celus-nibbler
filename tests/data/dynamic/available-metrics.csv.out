"2020-01-01","2020-01-31","","T1","Print_ISSN:1111-1111","","","","","Platform:P1|YOP:2015","deny","0"
"2020-02-01","2020-02-29","","T1","Print_ISSN:1111-1111","","","","","Platform:P1|YOP:2015","deny","1"
"2020-03-01","2020-03-31","","T1","Print_ISSN:1111-1111","","","","","Platform:P1|YOP:2015","deny","0"
"2020-01-01","2020-01-31","","T2","Print_ISSN:2222-2222","","","","","Platform:P1|YOP:2013","deny","3"
"2020-02-01","2020-02-29","","T2","Print_ISSN:2222-2222","","","","","Platform:P1|YOP:2013","deny","0"
"2020-03-01","2020-03-31","","T2","Print_ISSN:2222-2222","","","","","Platform:P1|YOP:2013","deny","2"
